use as npm module : npm install error

`npm install utec/geofrontend-server#master --save`

throws:

```
npm ERR! code ENOLOCAL
npm ERR! Could not install from "node_modules/geofrontend-server/basic-security-client" as it does not contain a package.json file.

```

--- a/package.json
+++ b/package.json
@@ -16,7 +16,6 @@
     "node": "8.9.4"
   },
   "dependencies": {
-    "basic-security-client": "file:basic-security-client",
     "cookie-parser": "1.4.4",
     "cors": "2.8.5",
     "express": "4.17.1",

